UserHelper.go

```diff
@@ -53,3 +55,13 @@ func FilterUsers(filter func(*User) bool) ([]*User, error) {
 
 	return filtered, nil
 }
+
+// SortUserByLastSeen sorts a list of users by their last seen date.
+func SortByLastSeen(users []*User) []*User {
```

Comment:
"Sort + plural" should be used. `SortUsersLastSeen` or `SortUsersByLastSeen` whichever you prefer.
In both the function and the comment.